.circleci/config.yml

```diff
@@ -713,10 +723,6 @@ workflows:
   # test, build and push all commits
   test-build-and-push:
     jobs:
-      - bake-gcp-dev-image:
```

Comment:
i think this got merged to master by accident. we should remove this from running on each commit